bin/xtc.sh

```diff
@@ -0,0 +1,104 @@
+source bin/log.sh
+
+set -e
+
+if [ -z ${1} ]; then
+  echo "Usage: ${0} device-set [DeviceAgent-SHA]
+
+Examples:
+
+$ bin/xtc.sh e9232255
+$ SKIP_IPA_BUILD=1 SERIES='Args and env' bin/xtc.sh e9232255
+$ SERIES='DeviceAgent 2.0' bin/xtc.sh e9232255 48d137d6228ccda303b2a71b0d09e1d0629bf980
+
+The DeviceAgent-SHA optional argument allows tests to be run against any
+DeviceAgent that has been uploaded to S3 rather than the current active
+DeviceAgent for Test Cloud.
+
+If you need to test local changes to run-loop or Calabash on Test Cloud,
+use the BUILD_RUN_LOOP and BUILD_CALABASH env variables.
+
+Responds to these env variables:
+
+        SERIES: the Test Cloud series
+SKIP_IPA_BUILD: iff 1, then skip re-building the ipa.
+                'make test-ipa' will still be called, so changes in the
+                features/ directory will be staged and sent to Test Cloud.
+BUILD_RUN_LOOP: iff 1, then rebuild run-loop gem before uploading.
+BUILD_RUN_LOOP: iff 1, then rebuild Calabash iOS gem before uploading.
```

Comment:
Can you add a variable here:

```
APP_LOCALE
```

Then add a `--locale` parameter to the `submit` command.

```
if [ "${APP_LOCALE}" = "" ]; then
  APP_LOCALE="en_US"
fi
```

```
--locale "${APP_LOCALE}"
```